src/writeData/subscriptions/utils/form.ts

```diff
@@ -0,0 +1,19 @@
+import {Subscription} from 'src/types/subscriptions'
+
+export const handleValidation = (
+  property: string,
+  formVal: string
+): string | null => {
+  if (!formVal) {
+    return `${property} is required`
+  }
+  return null
+}
+
+export const sanitizeForm = (form: Subscription): Subscription => {
+  if (form.brokerPassword === '' || form.brokerUsername === '') {
```

Comment:
should this be the other way around?
```suggestion
  if (form.brokerPassword !== '' || form.brokerUsername !== '') {
```

Reply:
ahhh I see. Got it, makes sense!